projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -317,6 +334,17 @@ export const getProductCardData = ( state, productSlug ) => {
 				productCardList: products.videopress ? products.videopress.features : [],
 				productCardIcon: '/recommendations/video-icon.svg',
 			};
+		case PLAN_JETPACK_BACKUP_T1_YEARLY:
+			return {
+				productCardTitle: __( 'Go back in time with one click', 'jetpack' ),
+				productCardCtaLink: getRedirectUrl( 'jetpack-recommendations-product-checkout', {
+					site: siteRawUrl,
+					path: productSlug,
+				} ),
+				productCardCtaText: __( 'Get Jetpack Backup', 'jetpack' ),
+				productCardList: products.backup ? products.backup.features : [],
```

Comment:
One of these features for backup has an asterisk on it due to a disclaimer we added to the product cards. 

![Screen Shot 2022-08-12 at 3 19 21 PM](https://user-images.githubusercontent.com/65001528/184446501-b8993258-7ff5-4701-86dc-0964b6b2e3cd.jpg)

The `products.backup` object should come with a `disclaimer` key that looks like this:

![image](https://user-images.githubusercontent.com/65001528/184446209-6fcfe469-ce2b-4090-a8ae-8f0da48e4099.png)

You may want to render this at the bottom of this card to avoid any confusion over a seemingly random asterisk. If it doesn't seem necessary here or you can't make it work with the design, perhaps we could filter out the asterisk in this case ( as a last resort 😄 )

Reply:
nice catch. I'll update that component to render the disclaimer text if it is provided.